Still incomplete, but we made huge progress

hugueslacroix opened the pull request:
The task is almost complete and it now works with the "to-be-merged" version of the cloze plugin. 

Linguistic background questionnaire is still disabled and incomplete, but the two other parts are almost final.

Commit:
Major update...

Commit:
updated readme

Commit:
modified readme and comments in the code.

Commit:
Added a lot of content in the task, and an incomplete style.css.

Commit:
Background questionnaire is getting almost ready

Commit:
almost complete...

Commit:
Updated with the latest form-plugin tab’s structure

Commit:
-Changed radios for checkboxes in language form.
-Modified instructions for audio-GJT
-Remove duplicated items (48-49) in the audio GJT.

Commit:
fix #3

Commit:
started coding the function to assign a code automatically.
adjusted CSS for likert scale, #6 fixed (must update jsPsych)
adjusted CSS for survey-text (must update jsPsych)
added a form with the possibility to enter a code previously assigned,
it allows the user to skip directly to the GJT task, #5 fixed.

Commit:
Modified required field, it should be better now (#2)

Commit:
uncommented first page and modified instructions

Commit:
Typos

@Maura04 corrected a few typos.

Commit:
Testing new structure with submodule

Commit:
New structure with submodule is now working, please note that you need to gulp the mdl lib before using it

Commit:
Moved readme file to root dir

Commit:
Many small fixes

Modified a few things in the CSS and in the form
Added Likert scale as required with a custom message in Spanish

rivasd commented:
Hey! no les he olvidados! solo que tengo mucho trabajo que acabar para el lunes proximo, despues podre verificar el codigo y integrar todo a Percept :)

Commit:
get update from jsPsych

Commit:
removed submodule mdl

Commit:
added distribution files from MDL library

Commit:
corrected a typo in the form

Commit:
added a few more infos to the data object

Also, instead of displaying the result, we save them as a CSV file.

Commit:
Added a way to save data on the pretest server (AJAX & PHP)

CSV files are stored in two different folders depending on the
code/group they were assigned.

Commit:
Removed stimuli length testing setting

Commit:
Added practice items

Added four practice items with feedback.

Commit:
added a few more special codes for native speaker who wants to pretest

Commit:
more codes

Commit:
more codes

Commit:
More codes

Commit:
error in code list

Commit:
MDL layout compatible!

Added intro page
Added consent form.
Added two codes: SKIP_TO_END, SKIP_PRACTICE.
Adapted items to mdl_layout.

Commit:
updated library

Commit:
updated lib

Commit:
updated lib

Commit:
Updated instructions and consent part.

Commit:
fixed participant code issue while saving file

Commit:
added loading

Commit:
added loading validation

Commit:
fixed loading

Commit:
added N26 to the code list.

Commit:
updated consent and final words

Commit:
added a few more correct options to the test

Commit:
Adjusted consent form

Commit:
Fixed many things in order to assign code automatically to participants

Codes are not stored externally.

Commit:
Temp files are now deleted when the task ends.

Commit:
Just added the task in french :)

Commit:
Updated french task

Commit:
changed v number in html

Commit:
again

Commit:
Updated translations and fixed a bug in saving files

Commit:
removed instructions in the two last pages.

Commit:
added numbers to french sentences.

Commit:
Code list is now loaded via php and is encrypted so the user can't see it in a plain XHR request

Commit:
Remove a console.log in the code form

Commit:
fixed a typo in french

Commit:
fixed other typos in french form

Commit:
again

Commit:
again ...

Commit:
changed instructions

Commit:
Added a way to resume tests

Added a prompt when user try to refresh the page during the test.

Commit:
little fix in resume code

Commit:
remove a console.log in conditional_function

Commit:
Minor changes

hugueslacroix committed:
Set theme jekyll-theme-slate
